engine/runtime/src/test/java/org/enso/interpreter/test/SignatureTest.java

```diff
@@ -944,13 +944,13 @@ public void returnTypeCheckOptInErrorZeroArgumentsExpression() throws Exception
             .buildLiteral();
 
     var module = ctx.eval(src);
-    var plusChecked = module.invokeMember(MethodNames.Module.EVAL_EXPRESSION, "foo");
-    assertEquals(8, plusChecked.execute(2).asInt());
+    var foo = module.invokeMember(MethodNames.Module.EVAL_EXPRESSION, "foo");
+    assertEquals(8, foo.execute(2).asInt());
     try {
-      var res = plusChecked.execute(".");
+      var res = foo.execute(".");
       fail("Expecting an exception, not: " + res);
     } catch (PolyglotException e) {
-      assertContains("expected `expression` to be Integer, but got Text", e.getMessage());
+      assertContains("expected `x` result to be Integer, but got Text", e.getMessage());
```

Comment:
Shouldn't this be `expecting 'x' to return Integer, but got Text`?

Reply:
Yes I think that form may be more readable, I will update it.